api/lib/domain/services/token-service.js

```diff
@@ -15,6 +15,15 @@ function createTokenForCampaignResults(userId) {
   }, settings.authentication.secret, { expiresIn: settings.authentication.tokenForCampaignResultLifespan });
 }
 
+function createTokenForStudentReconciliation({ userAttributes }) {
```

Comment:
Après tout, c'est une DS, donc un objet JS, le fait qu'il ait des attributs/propriétés est évident
```suggestion
function createTokenForStudentReconciliation({ user }) {
```
Ou alors, pour ne pas confondre avec un objet du domain (User), indiquer que ce sont des données reçues du GAR en utilisant le nom normalisé SAML
```suggestion
function createTokenForStudentReconciliation({ SAMLRequestAttributes }) {
```

Reply:
Le terme de `userAttributes` ne me dérange pas ici car ce sont bien des données relatives à un utilisateur (provenant du GAR) que nous passons en paramètre ici. Sinon ta deuxième proposition me semble pas mal non plus.